KappaAnalysis/interface/KappaSettings.h

```diff
@@ -102,6 +102,8 @@ class KappaSettings: public SettingsBase {
 
 
 	IMPL_SETTING(float, CrossSection);
```

Comment:
May add a default value of -1 here, too?
Else, this may brake if the CrossSection setting is not set.

Reply:
I think the cross section should always be set. Shouldn't it?

Reply:
Yes, I think it wouldn't make sense to run without giving a particular XS value. Is an error thrown, when the XS-value is not configured?

Reply:
Yes it should.
At the moment, we have `cfg['CrossSection'] = -1  # need to be set in individual config file` in the default config.
I think, a settings default would be more reasonable :D
(also for the planned default config cleanup)

Reply:
All right, I'll update it.